values.yaml

```diff
@@ -730,6 +730,14 @@ syncCatalog:
   # (Kubernetes -> Consul sync)
   k8sTag: null
 
+  # consulNodeName defines the Consul synthetic node that all services
+  # will be registered to.
+  # NOTE: Changing the node name and upgrading the Helm chart will leave
+  # all of the previously sync'd services registered with Consul and
+  # register them again under the new Consul node name. The out-of-date
+  # registrations will need to be explicitly removed.
```

Comment:
```suggestion
  # registrations will need to be manually removed.
```

Reply:
I'd originally phrased it this way, then wondered if it would stop folks from creating scripts to automate the cleanup process when necessary.

Reply:
That makes complete sense actually! 👍🏽